css/style.css

```diff
@@ -54,6 +54,268 @@ body {
   margin: auto;
 }
 
+header {
+  background-color:#033048;
+  float: left;
+  width: 100%;
+  padding-top: 10px;
+  padding-bottom: 20px;
+
+}
+
+header h1 {
+  font-weight: 700;
+font-size: 24px;
+color: #ffffff;
+padding-top: 10px;
+padding-left: 60px;
+padding-bottom: 10px;
+float: left;
+width: 20%;
+
+}
+
+.top-nav{
+  color: #f9e42e;
+  float: right;
+  width: 80%;
+  list-style-type: none;
+  text-align: center;
+  padding: 10px 10px 10px 10px;
+
+}
+
+.top-nav li {
+  float: right;
+  width: 10%;
+  text-align: center;
+  list-style-type: none;
+  padding-top: 10px;
+}
+
+.nav {
+  font-size: 14px;
+  color: #f9e42e ;
+  text-align: center;
+  font-weight: 300;
+}
+
+header a {
+  text-decoration: none;
+}
+
+.articles {
+  width: 73%;
+  float: left;
+
+}
+
+.article-header {
+  padding-right: 50px;
+}
+
+img {
+  max-width: 90%;
+  margin: auto;
+  padding-top: 20px;
+}
+
+/*.gutter {
+  float: left;
+  width: 20px;
+  height: 300px;
+  overflow: auto;
+}*/
+
+aside {
+  float: left;
+  width:27%;
+
+}
+
+main {
+  background-color:#f0efef;
+  padding-left: 40px;
+  padding-right: 40px;
+  padding-bottom: 100px;
+}
+
+article h2 {
+  font-size: 30px;
+  font-weight: 500px;
+  color: #033048;
+
+}
+  article h3 {
```

Comment:
Try to be consistent with indentation and spacing. It looks like there may be a stray space at the beginning of this line.